codecov.yml

```diff
@@ -1,12 +1,17 @@
 codecov:
-  token: 85c13d8a-e9e1-49fd-97d4-7ba103f3e830
   notify:
     wait_for_ci: no
   require_ci_to_pass: no
 coverage:
+  range: 75..98
```

Comment:
If a file has code coverage < 75%, it'll appear red.  Between 75% to 98%, it'll appear yellow.  Above 98%, it'll appear green.